bitvision/bitvision.py

```diff
@@ -15,126 +15,160 @@
 CONFIG = configparser.ConfigParser()
 CONFIG.read("config.ini")
 
-
 #########
 # HELPERS
 #########
 
+# BitVision Command Line Trading Platform
+# Aaron Lichtman and Jon Shobrook
 
-def login():
+import os
+import sys
+import urwid
+
+# Place Your Own Trade is not shown until Trading Deactivated.
+choices = ["Activate/Deactivate Automated Trading", "Deposit/Withdraw Money", "Place Your Own Trade"]
```

Comment:
I'd like to set this up so that the menu shows "Automated Trading: ON" or "Automated Trading: OFF" depending on its current state, instead of showing "ON/OFF". This will make it more clear to the user what state the system is in.

Reply:
"Deposit/Withdraw Money" should be separated into "Deposit Money" and "Withdraw Money".

Reply:
I removed this block of code in favor of the footer menu bar.